docs/extensions.rst

```diff
@@ -133,3 +133,37 @@ TLS
 ---
 
 See :doc:`specs/tls`.
+
+Early hints
+-----------
+
+An informational response with the status code 103 is an early hint,
```

Comment:
Can we have Early Hint either capitalised or in quotes here to indicate it is the proper name of the HTTP feature?

Reply:
Yep, done